Directory.Packages.props

```diff
@@ -1,41 +1,42 @@
 <Project>
-	<PropertyGroup>
-		<ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-		<AvaloniaVersion>11.0.5</AvaloniaVersion>
-	</PropertyGroup>
-	<ItemGroup>
-		<!-- JSON. -->
-		<PackageVersion Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="7.0.9" />
-		<!-- SQLite. -->
-		<PackageVersion Include="Microsoft.Data.Sqlite" Version="7.0.9" />
-		<PackageVersion Include="Microsoft.Win32.SystemEvents" Version="7.0.0" />
-		<!-- C# analysis. -->
-		<PackageVersion Include="Microsoft.CodeAnalysis.Common" Version="4.6.0" />
-		<PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.6.0" />
-		<PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" />
-		<!-- Core libraries. -->
-		<PackageVersion Include="WabiSabi" Version="1.0.1.2" />
-		<PackageVersion Include="NBitcoin" Version="7.0.27" />
-		<!-- Backend. -->
-		<PackageVersion Include="Swashbuckle.AspNetCore" Version="6.5.0" />
-		<!-- UI. -->
-		<PackageVersion Include="Avalonia" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="Avalonia.Controls.TreeDataGrid" Version="11.0.1" />
-		<PackageVersion Include="Avalonia.Diagnostics" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="Avalonia.Xaml.Behaviors" Version="11.0.2" />
-		<PackageVersion Include="Avalonia.Desktop" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="Avalonia.ReactiveUI" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="Avalonia.Skia" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="Avalonia.Themes.Fluent" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="Avalonia.Fonts.Inter" Version="$(AvaloniaVersion)" />
-		<PackageVersion Include="System.Runtime" Version="4.3.1" />
-		<PackageVersion Include="QRackers" Version="1.1.0" />
-		<!-- Testing. -->
-		<PackageVersion Include="coverlet.collector" Version="6.0.0" />
-		<PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.9" />
-		<PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.6.3" />
-		<PackageVersion Include="Moq" Version="[4.18.4]" />
-		<PackageVersion Include="xunit" Version="2.5.0" />
-		<PackageVersion Include="xunit.runner.visualstudio" Version="2.5.0" />
-	</ItemGroup>
-</Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <AvaloniaVersion>11.0.5</AvaloniaVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <!-- JSON. -->
+    <PackageVersion Include="DynamicData" Version="8.1.1" />
```

Comment:
I think it shouldn't be under `JSON` comment

Reply:
Fixed.